src/scss/objects/_form-toggle.scss

```diff
@@ -33,7 +38,7 @@ $formToggle-text--checked           : $color-green;
         display: inline-block;
         margin-bottom: spacing();
         border-radius: $formToggle-border-radius;
-        padding: ($formToggle-padding / 2) $formToggle-padding;
+        padding: math.div($formToggle-padding, 2) $formToggle-padding;
```

Comment:
Would it be worth having separate variables for vertical and horizontal padding?

Reply:
The eventual aim for this will be to deprecate these optional components as they move over to be fozzie-components. If we ever do need to make them different padding sizes though, can always split this out at that point potentially.